core/src/main/java/com/linecorp/armeria/client/BlockingWebClient.java

```diff
@@ -0,0 +1,372 @@
+        return execute(RequestHeaders.of(HttpMethod.GET, path));
+    }
+
+    /**
+     * Sends an HTTP GET request, accepts query parameters to append to the path.
+     */
+    default AggregatedHttpResponse get(String path, QueryParams params) {
+        return execute(RequestHeaders.of(HttpMethod.GET, WebClientUtil.addQueryParams(path, params)));
+    }
+
+    /**
+     * Sends an HTTP HEAD request.
+     */
+    default AggregatedHttpResponse head(String path) {
+        return execute(RequestHeaders.of(HttpMethod.HEAD, path));
+    }
+
+    /**
+     * Sends an HTTP HEAD request, accepts query parameters to append to the path.
+     */
+    default AggregatedHttpResponse head(String path, QueryParams params) {
+        return execute(RequestHeaders.of(HttpMethod.HEAD, WebClientUtil.addQueryParams(path, params)));
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content.
+     */
+    default AggregatedHttpResponse post(String path, HttpData content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, path), content);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content, accepts query parameters to append to the path.
+     */
+    default AggregatedHttpResponse post(String path, QueryParams params, HttpData content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, WebClientUtil.addQueryParams(path, params)), content);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content.
+     */
+    default AggregatedHttpResponse post(String path, byte[] content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, path), content);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content, accepts query parameters to append to the path.
+     */
+    default AggregatedHttpResponse post(String path, QueryParams params, byte[] content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, WebClientUtil.addQueryParams(path, params)), content);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content.
+     */
+    default AggregatedHttpResponse post(String path, String content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, path), HttpData.ofUtf8(content));
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content, accepts query parameters to append to the path.
+     */
+    default AggregatedHttpResponse post(String path, QueryParams params, String content) {
+        return execute(RequestHeaders.of(HttpMethod.POST, WebClientUtil.addQueryParams(path, params)), content);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content.
+     */
+    default AggregatedHttpResponse post(String path, String content, Charset charset) {
+        return execute(RequestHeaders.of(HttpMethod.POST, path), content, charset);
+    }
+
+    /**
+     * Sends an HTTP POST request with the specified content, accepts query parameters to append to the path.
```

Comment:
```suggestion
     * Sends an HTTP POST request with the specified content, appending the given query parameters to the path.
```
Please fix them all.